src/domain/module-file.ts

```diff
@@ -0,0 +1,36 @@
+import fs from "node:fs";
+
+export class ModuleFile {
+  private moduleContent: string;
+
+  constructor(filePath: string) {
+    this.moduleContent = fs.readFileSync(filePath, "utf8");
+  }
+
+  public get moduleName(): string {
+    const regex = /module\([^)]*?name\s*=\s*"(\w+)"/s;
```

Comment:
kinda gross, I assumed you already considered other ways to parse the file content? I guess on GH actions it's hard to bring any other language runtime and this is a simple enough regex

Reply:
Yeah I didn't want to get into parsing `.bazel`/`.bzl` files.

Reply:
note that `-` and `.` are valid characters in module names: https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/cmdline/RepositoryName.java;l=54;drc=07c5c1aa6d0b63605ae793dce78d26122af64a84

(needs better documentation than code, I know...)

Reply:
Good catch. I used that regex, added a link, and a test.